multimod/internal/prerelease/prerelease.go

```diff
@@ -203,21 +203,21 @@ func (p prerelease) updateAllGoModFiles() error {
 	return nil
 }
 
-func (p prerelease) commitChangesToNewBranch(repo *git.Repository) error {
-	branchNameElements := []string{"prerelease", p.ModuleSetRelease.ModSetName, p.ModuleSetRelease.ModSetVersion()}
-	branchName := strings.Join(branchNameElements, "_")
-
-	commitMessage := fmt.Sprintf(
-		"Prepare %v for version %v",
-		p.ModuleSetRelease.ModSetName,
-		p.ModuleSetRelease.ModSetVersion(),
-	)
-
-	hash, err := common.CommitChangesToNewBranch(branchName, commitMessage, repo, nil)
+func commitChanges(msr common.ModuleSetRelease, commitToDifferentBranch bool, repo *git.Repository) error {
+	commitMessage := fmt.Sprintf("Prepare %v for version %v", msr.ModSetName, msr.ModSetVersion())
+
+	var hash plumbing.Hash
+	var err error
+	if !commitToDifferentBranch {
```

Comment:
Is this logic inverted?

```suggestion
	if commitToDifferentBranch {
```